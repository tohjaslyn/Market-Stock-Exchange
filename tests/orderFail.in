order 0000
add trader1 100
buy trader1 ABC 10 1.0
order 0001
exit
