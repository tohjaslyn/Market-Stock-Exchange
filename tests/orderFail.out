$ No orders in either book in the market.
$ Success.
$ No trades could be made, order added to buy book.
$ Order is not present in either order book.
$ Have a nice day.
